buildout.cfg

```diff
@@ -26,6 +30,7 @@ eggs =
     Plone
     Pillow
     redturtle.agidtheme [test]
+#    plone.app.mosaic
```

Comment:
If it is not required and commented, we should remove this before merge

Reply:
completely agree @nzambello; 
done!